File: Tutorial4-6SubComponentBuildersMultipleModulesAndActvities/src/main/java/com/example/tutorial4_6subcomponentbuildersmultiplemodulesandactvities/model/ToastMaker.java
package com.example.tutorial4_6subcomponentbuildersmultiplemodulesandactvities.model;

import android.content.Context;
import android.widget.Toast;

public class ToastMaker {

    private Context context;

    public ToastMaker(Context context) {
        this.context = context;
    }

    public void showToast(String message) {
        Toast.makeText(context, message, Toast.LENGTH_SHORT).show();
    }


}

File: Tutorial5-3Dagger-Android-MultipleActivitiesAndFragments/src/main/java/com/example/tutorial5_3dagger_android_multipleactivitiesandfragments/model/ToastMaker.kt
package com.example.tutorial5_3dagger_android_multipleactivitiesandfragments.model

import android.content.Context
import android.widget.Toast

class ToastMaker(private val context: Context) {

    init {
        println("🎃 ToastMaker init context: $context")
    }

    fun showToast(message: String) {

        Toast.makeText(context, message, Toast.LENGTH_SHORT).show()
    }

}